gallery_get.py

```diff
@@ -37,6 +37,7 @@
 QUEUE = queue.Queue()
 STANDBY = False
 THREADS = []
+MAX_THREADS = 10

```

Comment:
What if instead of a static variable, the Plugin class had a new member called max_threads? That way, if the problem we're solving is that certain gallery sites block on thread counts, we can throttle just for those sites.

For example: start_jobs() is called by add_job(), and add_job() takes a Plugin as a parameter... so if max_threads was a Plugin member you could pass that value into start_jobs().  Granted, this assumes one gallery site for an execution for gallery_get (which isn't necessarily the case when calling from reddit_get), but I think that's fine.

Reply:
Actually that is an even better idea! I’ll work on that and submit a new PR!